hack/go-mod.sh

```diff
@@ -16,9 +16,6 @@
 
 . ./hack/go-mod-env.sh
 
-echo Resetting vendor directory
-
 setGoModEnv
 
 go mod tidy
-go mod vendor
```

Comment:
No more vendor on go 1.13?

Reply:
We don’t need it any more. And operator SDK has also removed dep support.